lib/smart_answer_flows/marriage-abroad/outcomes/_services_and_fees.govspeak.erb

```diff
@@ -12,14 +12,15 @@
      calculator.ceremony_country_is_french_overseas_territory? ||
      calculator.ceremony_country == 'indonesia' ||
      calculator.ceremony_country == 'monaco' ||
-     calculator.ceremony_country == 'france' %>
+     calculator.ceremony_country == 'france' ||
+     calculator.ceremony_country == 'cambodia' ||
+     (calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?) %>
 
     <% if calculator.ceremony_country == 'vietnam' %>
       <%= render partial: 'link_to_consular_fees_pay_in_sterling_or_local_currency.govspeak.erb',
                  locals: { calculator: calculator } %>
 
-    <% elsif calculator.ceremony_country != 'cambodia' &&
-      !(calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?) %>
+    <% else %>
```

Comment:
Minor: It might help to point out in the commit note that you are moving the conditions from an `if` statement to an `unless` statement and thus why:

```ruby
calculator.ceremony_country != 'cambodia' &&
  !(calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?)
```
has become:
```ruby
calculator.ceremony_country == 'cambodia' ||
  (calculator.ceremony_country == 'brazil' && calculator.resident_outside_of_third_country?)
```